packages/core/src/scp-cf/client-credentials-token-cache.ts

```diff
@@ -21,7 +22,11 @@ const ClientCredentialsTokenCache = (
     credentials: ClientCredentials,
     token: ClientCredentialsResponse
   ): void => {
-    cache.set(getGrantTokenCacheKey(url, credentials), token, token.expires_in);
+    cache.set(
+      getGrantTokenCacheKey(url, credentials),
+      token,
+      moment().add(token.expires_in, 'second').unix() * 1000
```

Comment:
A typical `expires_in` value is `43199`, which means 12 hours later it will expire.